### ISSUE TITLE
Write validation checks for source inventories

### ISSUE THREAD

**Issue by edwardsmarc** [edwardsmarc/CASFRI#262]
One element of CASFRI 05 is to report back to data providers on errors in their data. We can validate source data using the validation functionality of the engine. This involves writing a translation table with validation functions that run on the source data and reports errors in a log file.

This issue will record a list of validations we may want to produce:

1. Layer ids. Check that all layer ids that we expect to have values, do in fact have values. For example a forested polygon in BC should always have a layer_id value.

**Comment by pedrogit** [comment]
Isn't that what the logging is already doing?

**Comment by edwardsmarc** [comment]
Yes, we but don't use layer_id in any of our translations so it doesn't get validated. This issue would be applying the same method we use to validate photo year and species tables using e.g. ab_avi01_species_validation.csv and ab_photo_year_validation.csv.
